.github/workflows/update-prettier.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: lts/*
-      - uses: actions/cache@fd5de65bc895cf536527842281bea11763fefd77
+      - uses: actions/cache@12681847c623a9274356751fdf0a63576ff3f846
```

Comment:
we should stop using `actions/cache`, and instead add `cache: npm` to the step above